examples/rt-for-vmware.yml

```diff
@@ -0,0 +1,36 @@
+kernel:
+  image: linuxkit/kernel:4.14.18-rt
+  cmdline: "console=tty0"
+init:
+  - linuxkit/init:v0.2
+  - linuxkit/runc:v0.2
+  - linuxkit/containerd:v0.2
+  - linuxkit/ca-certificates:v0.2
+onboot:
+  - name: sysctl
+    image: linuxkit/sysctl:v0.2
+services:
+  - name: getty
+    image: linuxkit/getty:v0.2
+    env:
+     - INSECURE=true
+  - name: rngd
+    image: linuxkit/rngd:v0.2
+  - name: dhcpcd
+    image: linuxkit/dhcpcd:v0.2
+  - name: open-vm-tools
+    image: linuxkit/open-vm-tools:30b93dde3bbcc2346f01a570b932b768b04fc54c
```

Comment:
Is there a reason you are not using the `v0.2` tag here?

Reply:
@rn Will set that as v0.2.